custom_components/mikrotik_router/mikrotikapi.py

```diff
@@ -482,3 +482,106 @@ def get_traffic(self, interfaces) -> Optional(list):
 
         self.lock.release()
         return traffic if traffic else None
+
+    @staticmethod
+    def _current_milliseconds():
+        from time import time
+        return int(round(time() * 1000))
+
+    def is_accounting_enabled(self) -> bool:
+        accounting = self.path("/ip/accounting", return_list=True)
```

Comment:
return_list=True is now default. Since that used the most. only return_list=False have to be specified